training/src/anemoi/training/train/train.py

```diff
@@ -165,12 +165,10 @@ def model(self) -> pl.LightningModule:
             "supporting_arrays": self.supporting_arrays,
         }
 
-        train_module = importlib.import_module(
-            getattr(self.config.training, "train_module", "anemoi.training.train.forecaster"),
-        )
-        train_func = getattr(train_module, getattr(self.config.training, "train_function", "GraphForecaster"))
-        model = train_func(**kwargs)
-        # NOTE: This should be replaced with instantiate
+        module_name, class_name = self.config.training.task.rsplit(".", 1)
```

Comment:
Can this be replaced with a `get_class` 
see https://github.com/ecmwf/anemoi-core/blob/d3bf2ae37bb38d5e9b85626a295eaa0b7b8b6f3a/training/src/anemoi/training/train/train.py#L185